labs/lab05-airgap/README.md

```diff
@@ -262,7 +261,7 @@ Again we'll use `REPLICATED_APP` to construct the DNS name but you can input it
 ```shell
 export JUMP_BOX_IP=... # your jumpbox IP
 export REPLICATED_APP=... # your app slug
-ssh -NL 8800:${REPLICATED_APP}-lab05-airgap:8800 -L 8888:${REPLICATED_APP}-lab05-airgap:8888 kots@${JUMP_BOX_IP}
+ssh -NL 8800:${REPLICATED_APP}-lab05-airgap:8800 -L 8888:${REPLICATED_APP}-lab05-airgap:8888 <your_first_name>@${JUMP_BOX_IP}
```

Comment:
for commands where there is user input, could we follow the pattern of `export YOUR_NAME=...` and then use that environment variable in the commands -- that way users can copy paste directly without having to edit each one